x-pack/test/reporting_api_integration/reporting_and_security/validation.ts

```diff
@@ -43,7 +43,7 @@ export default function ({ getService }: FtrProviderContext) {
           { jobParams: createPdfV2Params(1541.5999755859375) }
         );
 
-        await retry.tryForTime(18000, async () => {
+        await retry.tryForTime(48000, async () => {
```

Comment:
When this test runs can you tell how long it actually took?  And did something about the changes in this PR make it take longer?

Reply:
I can play with milliseconds and find out.  As far as the root cause, I'm not sure currently.

Reply:
@LeeDr 
```
[09:24:29.685384000] │ debg ReportingAPI.postJobJSON((/api/reporting/generate/printablePdfV2): {"jobParams":"(browserTimezone:UTC,layout:(dimensions:(height:1492,width:1541.5999755859375),id:preserve_layout),locatorParams:!((id:DASHBOARD_APP_LOCATOR,params:(dashboardId:'6c263e00-1c6d-11ea-a100-8589bb9d7c6b',preserveSavedFilters:!t,timeRange:(from:'2019-03-23T03:06:17.785Z',to:'2019-10-04T02:33:16.708Z'),useHash:!f,viewMode:view),version:'8.2.0')),objectType:dashboard,title:'Ecom Dashboard',version:'8.2.0')"})
[09:24:32.419050000] │ debg Report at path /api/reporting/jobs/download/l60hgmw40jm19d0062fre725 is pending
[09:24:32.425963000] │ debg --- retry.tryForTime error: expected 503 to equal 200
[09:24:32.933635000] │ debg Report at path /api/reporting/jobs/download/l60hgmw40jm19d0062fre725 is pending
[09:24:50.519635000] │ debg --- retry.tryForTime failed again with the same message...
[09:24:51.026097000] │ debg Report at path /api/reporting/jobs/download/l60hgmw40jm19d0062fre725 is pending
[09:24:51.031005000] │ debg --- retry.tryForTime failed again with the same message...
[09:24:51.580188000] │ debg Report at path /api/reporting/jobs/download/l60hgmw40jm19d0062fre725 is complete
[09:24:51.586429000] └- ✓ pass  (21.9s)
```

Ok, so after using your timestamp alias, I've decreased the timeout to 30Kms

Reply:
after chatting with @LeeDr we are gonna leave at the long timeout.

Reply:
+1 We don't want the test to be flaky on Cloud where I think there is more variability in timing.